ci/scripts/build.sh

```diff
@@ -175,7 +175,4 @@ if [[ ${COMPILE_BUILD} == "ON" ]];then
     fi
 fi
 
-echo -e "===\n=== ccache statistics after build\n==="
```

Comment:
I prefer to keep this, show ccache hit rate after build

Reply:
See `ci/scripts/update_cache.sh`